packages/alfa-json/src/serializable.ts

```diff
@@ -20,6 +20,12 @@ export interface Serializable<T extends JSON = JSON> {
 }
 
 export namespace Serializable {
+  export type ToJSON<T> = T extends Serializable<infer T>
```

Comment:
Shouldn't the type be called `JSON<T>` rather than `ToJSON<T>`?
As far as I can tell, this is the type of serialised objects (JSON), not the type of serialising function (to JSON).

Reply:
It's basically the type-level equivalent of the `Serializable.toJSON()` function as it transforms a type `T` into its corresponding JSON serialised type.